core/core-keybindings.el

```diff
@@ -131,6 +131,20 @@ minor-mode, the third argument should be non nil."
               :evil-states (normal motion visual evilified)))
           (boundp prefix)))))
 
+(defun spacemacs/inherit-leader-keys-from-parent-mode (mode &optional parent-mode)
+  "Make derived mode MODE inherit leader key bindings from PARENT-MODE.
+
+If omitted, PARENT-MODE defaults to the parent mode of MODE.
+Signal an error if MODE is not a derived mode."
```

Comment:
You can delete the blank line on L136. Otherwise there'll be two blank lines.

It's worth mentioning that if `mode` itself is nil, as in the case when this function is called before the package is loaded, then we cannot derive the parent mode as well.

So just add a note in docstring that people should call this function after the package is loaded.

Reply:
I added a note mentioning the package loading thing, but I'm a bit confused about your note about the blank line.  I didn't see an extra blank line in describe-function, but I also saw the same result with the blank line removed, so I went ahead and implemented your suggestion to be consistent with the rest of the file.

Reply:
Oh sorry. There won't be two blank lines.

It just automatically adds a blank line after first sentence.
This is a tacit behavior.